pom.xml

```diff
@@ -403,18 +403,17 @@
         </profile>
     </profiles>
     <properties>
-        <vertx.version>4.2.1</vertx.version>
-        <vertx-redis-client.version>4.3.3</vertx-redis-client.version>
-        <slf4j.version>1.7.32</slf4j.version>
-        <junit.version>4.13.1</junit.version>
+        <vertx.version>4.5.1</vertx.version>
```

Comment:
Do we plan to also update the version in [gateleen](https://github.com/swisspost/gateleen) and all other friends? Is this basically the first step to generally make the upgrade?

Reply:
Yes I am currently working on this. All the friends are prepared (with open PRs).